packages/astro/src/runtime/client/dev-toolbar/apps/audit/rules/a11y.ts

```diff
@@ -125,12 +125,6 @@ const a11y_required_content = [
 
 const a11y_distracting_elements = ['blink', 'marquee'];
 
-// Unused for now
```

Comment:
> for now

These are almost half a year old. May I suggest removing them? 🙂

Reply:
@Princesseuh do we want to keep them as comments for future usage or something?

Reply:
Nah, it's fine to remove.